PeerioTechnologies/peerio-desktop
# WIP: beacons use Provider/inject instead of singleton store

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/15152/desktop-refactor-beacons-move-from-ui-store-to-context-pattern

#### Testing instructions
Beacons should work the same as before, no change on the user side. Beacons that are currently on desktop:
* Welcome screen
  * spot beacon on Contacts icon
* Invite contact screen
  * area beacon on search bar
  * spot beacon on Files icon
* Files screen
  * area beacon on Upload button
  * spot beacon on Chats icon
* Chats screen
  * area beacon on plus icon of "Direct messages" button

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
